src/util/utils.js

```diff
@@ -628,3 +630,31 @@ export const getYesterdayDateRoundDownHour = () => {
   const yesterday = date.setDate(date.getDate() - 1)
   return new Date(yesterday).toISOString()
 }
+
+export const getMonthName = (month: number, short: boolean = false) => {
```

Comment:
This does not return localized formats or month names. Use IntlDateTimeFormat() instead. This should be placed in Intl.js

Reply:
Refactored